packages/plugin-dlx/sources/commands/dlx.ts

```diff
@@ -43,7 +43,29 @@ export default class DlxCommand extends BaseCommand {
 
       await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`package.json`)), `{}\n`);
       await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`yarn.lock`)), ``);
-      await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`.yarnrc.yml`)), `enableGlobalCache: true\n`);
+
+      const projectCwd = (await Configuration.findProjectCwd(this.context.cwd, Filename.lockfile))!;
```

Comment:
You can't assume that `projectCwd` will always be there - `yarn dlx` may be executed outside of a project context. Very rare, but better just add a condition to guard against that 🙂